packages/help-center/src/components/help-center-smooch.tsx

```diff
@@ -186,21 +202,38 @@ const HelpCenterSmooch: React.FC< { enableAuth: boolean } > = ( { enableAuth } )
 			setZendeskClientId( getClientId( allConversations ) );
 			Smooch.on( 'message:received', getUnreadListener );
 			Smooch.on( 'message:sent', clientIdListener );
-			Smooch.on( 'disconnected', () => {
-				recordTracksEvent( 'calypso_smooch_messenger_disconnected' );
-			} );
-			Smooch.on( 'reconnecting', () => {
-				recordTracksEvent( 'calypso_smooch_messenger_reconnecting' );
-			} );
+			Smooch.on( 'disconnected', disconnectedListener );
+			Smooch.on( 'reconnecting', reconnectingListener );
+			Smooch.on( 'connected', connectedListener );
 		}
 
 		return () => {
 			// @ts-expect-error -- 'off' is not part of the def.
 			Smooch?.off?.( 'message:received', getUnreadListener );
 			// @ts-expect-error -- 'off' is not part of the def.
 			Smooch?.off?.( 'message:sent', clientIdListener );
+			// @ts-expect-error -- 'off' is not part of the def.
+			Smooch?.off?.( 'disconnected', disconnectedListener );
+			// @ts-expect-error -- 'off' is not part of the def.
+			Smooch?.off?.( 'reconnecting', reconnectingListener );
+			// @ts-expect-error -- 'off' is not part of the def.
+			Smooch?.off?.( 'connected', connectedListener );
+			// @ts-expect-error -- 'off' is not part of the def.
+			Smooch?.off?.( 'message:received', getUnreadListener );
+			// @ts-expect-error -- 'off' is not part of the def.
+			Smooch?.off?.( 'message:sent', clientIdListener );
```

Comment:
This is duplicated, or am I missing something? @alshakero @heavyweight

Reply:
yes
could be something that was brought via merge or rebase
I can see duplications for `getUnreadListener` and `clientIdListener`

Reply:
https://github.com/Automattic/wp-calypso/pull/105652 👍